server/src/main/java/org/candlepin/service/impl/HypervisorUpdateAction.java

```diff
@@ -204,6 +202,21 @@ else if (jobReporterId == null) {
         return new Result(result, hypervisorKnownConsumersMap);
     }
 
+    private Consumer reconcileByUuid(Consumer knownHost, Consumer incoming, Map<String,
+        Consumer> systemUuidKnownConsumersMap) {
```

Comment:
Probably better for readability to not break the `Map<String,Consumer>` in half with line break (I'm actually surprised checkstyle didn't complain)